sdk/core/Azure.Core/api/Azure.Core.net462.cs

```diff
@@ -594,6 +594,7 @@ protected RequestContent() { }
         public static Azure.Core.RequestContent Create(object serializable, Azure.Core.Serialization.ObjectSerializer? serializer) { throw null; }
         public static Azure.Core.RequestContent Create(System.ReadOnlyMemory<byte> bytes) { throw null; }
         public static Azure.Core.RequestContent Create(string content) { throw null; }
+        public static Azure.Core.RequestContent Create<T>(T model, System.ClientModel.Primitives.ModelReaderWriterOptions? options = null) where T : System.ClientModel.Primitives.IPersistableModel<T> { throw null; }
```

Comment:
We shouldn't be manually modifying this file. You can regenerate the API by running `eng/scripts/Export-Api.ps1 core`.